redhat-cop/agnosticd
# add 2.3.0 to supported versions

## Description

See additional info/commit messages. Verification for 5936a2e77df91cfadc395aaf1ecde040d4045e84 followed and works as expected against an OpenShift 4.x cluster

## Additional Info

commit 5936a2e77df91cfadc395aaf1ecde040d4045e84

add a new supported version, 2.3.0 to the list of supported
versions of rhmi 2.x in the ocp4_workload_integreatly role

components:
- role, ocp4_workload_integreatly

verification:
- ensure install completes against an openshift 4.x cluster using
  the new default values
- ensure 2.2.0 still installs as expected

commit ee8bf042c63ea28cbb60a8142710b8caec0e22c7

add aiden keating <aidenkeating> as a named author to the rhmi
2.x role instead of the more generic team
